packages/tools/atf/package.mk

```diff
@@ -15,7 +15,15 @@ PKG_TOOLCHAIN="manual"
 [ -n "$KERNEL_TOOLCHAIN" ] && PKG_DEPENDS_TARGET+=" gcc-arm-$KERNEL_TOOLCHAIN:host"
 
 make_target() {
-  CROSS_COMPILE="$TARGET_KERNEL_PREFIX" LDFLAGS="" CFLAGS="" make PLAT=$ATF_PLATFORM bl31
+  ATF_OPTS="PLAT=$ATF_PLATFORM"
+
+  # atf gained support GCC stack protection, but it's currently not implemented
+  # on platforms supported by LE
+  #if flag_enabled "hardening" "$HARDENING_SUPPORT"; then
```

Comment:
Can you uncomment this section. If hardening is opt in then we can check for failures here when we come across it.

Reply:
Unfortunately not, if I uncomment that atf fails to build for sunxi when I enable hardening locally